dlc/src/main/scala/org/bitcoins/dlc/testgen/DLCTestVector.scala

```diff
@@ -357,11 +378,27 @@ object SerializedDLCTestVectorSerializers {
       json.validate[String].map(factory.fromHex)
   }
 
-  implicit val transactionOutPointReads: Reads[TransactionOutPoint] = hexReads(
-    TransactionOutPoint)
-  implicit val transactionOutputReads: Reads[TransactionOutput] = hexReads(
-    TransactionOutput)
-  implicit val extPrivKeyReads: Reads[ExtPrivateKey] = hexReads(ExtPrivateKey)
+  implicit val ecPublicKeyReads: Reads[ECPublicKey] = hexReads(ECPublicKey)
```

Comment:
unrelated, but i think we should be moving serializers like these into `db-commons` (which should be renamed). Aren't these already implemented for the bitcoind-rpc project? https://github.com/bitcoin-s/bitcoin-s/blob/3a5af7e3ebcbd7dbe535c08b79ddda9a4a5612c0/bitcoind-rpc/src/main/scala/org/bitcoins/rpc/serializers/JsonSerializers.scala#L35

Reply:
#1043